frontend/src/components/AddRecipeForm.js

```diff
@@ -1,11 +1,33 @@
 import { useFormik } from 'formik';
 import axios from 'axios';
 import { useEffect, useState } from 'react';
+import * as yup from 'yup';
+
+const validationSchema = yup.object().shape({
+  name: yup
+    .string()
+    .required('A felhasználónév kötelező!')
+    .min(10, 'Minimum 10 karakter.'),
+  description: yup
+    .string()
+    .required('A leírés kötelező!')
+    .min(10, 'Minimum 10 karakter.'),
+  preparationTime: yup
+    .number('Nem szám formátum')
+    .required('Kötelező mező')
+    .max(1000, 'Túl sok')
+    .integer('Szám formátum szükséges '),
+});
 
 const showAlert = () => {
   alert('Sikeres küldés!');
```

Comment:
Majd ezek az alertek helyett valami igényesebb megjelenésű visszajelzést is kaphatna a felhasználó.

Reply:
Szívesen veszek bármilyen tanácsot , sajnos nem vagyok gyakorlott ilyenben, próbáltam valami visszajelzést adni. De ha jól emlékszem volt ilyen az egyik elméleti órán regisztráció kezelésére, este majd újra nézem.

Reply:
Csak valami HTML-ből épített popupra vagy modalra gondoltam, pl: https://getbootstrap.com/docs/4.0/components/modal/

Reply:
De az alert is jobb visszajelzés a semminél! Beszéljétek meg reggel a PM-mel, hogy érdemes-e fancyzni vagy sem.

Reply:
Modal hozzáadva  és kiszervezve uj komponensbe